### DISCUSSION TITLE
PaddleOCR一直提示离线数据不完整

### DISCUSSION THREAD

**Discussion by Johnnsy1024** [STranslate/STranslate#132] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ba51de1d-21e2-4ed3-addc-9153138c74f4)
![image](https://github.com/user-attachments/assets/8912209a-60f2-4982-ac9c-9158bc3e6532)

**Comment by Johnnsy1024** [comment]
下载数据包能下载成功并加载，但是不起作用

**Reply by ZGGSONG** [reply, marked as answer]
你是不是使用的管理员权限启动的软件再下载的数据包，后面用普通用户权限打开软件的，如果是，请清理ocr数据包后使用普通用户权限下载
> 这个暂时我复现不了，但有的人这样解决了

如果不是的话，建议直接退出软件后运行软件目录下的`cleanocr.bat`文件，清理ocr数据包后直接打开软件重新下载再试试

**Reply by Johnnsy1024** [reply]
谢谢，运行`cleanocr.bat`文件是可以解决的👍
